pkg/util/useragent.go

```diff
@@ -22,16 +22,18 @@ type CentralUserAgent struct {
 	IsGRPC              bool   `json:"reactive"`
 	HostName            string `json:"hostname,omitempty"`
 	UseGRPCStatusUpdate bool   `json:"-"`
+	RuntimeID           string `json:"runtimeId,omitempty"`
```

Comment:
the regexps at the top need some updating

Reply:
during my added tests, it seems that there are no current tests for the option of grpc (as part of the regex) - ??(grpc-go.*\/\d*.\d*.\d*)?$`).  I don't believe ?? is actual valid, no?

Reply:
resolving this for a better regex provided by jason